mobile/src/main/res/values/dimens.xml

```diff
@@ -10,4 +10,5 @@
     <dimen name="widgetlist_item_left_margin">0dp</dimen>
     <dimen name="widgetlist_item_right_margin">0dp</dimen>
     <dimen name="widget_margin">0dp</dimen>
+    <dimen name="webview_row_height">36dp</dimen>
```

Comment:
I'm open for suggestions regarding this value. BasicUI uses 36 pixels per row, which is what I based this on. Old code used 80 pixels per row, which amounts to either 80dp (mdpi), ~53dp (hdpi), 40dp (xhdpi) and ~27dp (xxhdpi). Most devices should be xxhdpi nowadays.